src/core/common/new.hpp

```diff
@@ -39,6 +39,6 @@
 
 inline void *operator new(size_t, void *p) throw() { return p; }
 
-void OT_CDECL operator delete(void *, size_t) throw();
+void OT_CDECL operator delete(void *) throw();
```

Comment:
The Windows kernel mode code is looking for the sized operator. I'm not sure why. Can we just add both?